src/subscription_manager/release.py

```diff
@@ -81,22 +81,27 @@ def __init__(self):
     def get_releases(self):
         # cdn base url
 
-        # find the rhel product
-        release_product = None
+        # Find the rhel products
+        release_products = []
         installed_products = self.product_dir.get_installed_products()
         for product_hash in installed_products:
             product_cert = installed_products[product_hash]
             products = product_cert.products
             for product in products:
                 rhel_matcher = rhelproduct.RHELProductMatcher(product)
                 if rhel_matcher.is_rhel():
-                    release_product = product
+                    release_products.append(product)
 
-        if release_product is None:
-            log.debug("No products with RHEL product tags found")
+        if len(release_products) == 0:
+            log.info("No products with RHEL product tags found")
             return []
+        elif len(release_products) > 1:
+            raise ValueError("More then one product with RHEL product tags found.")
```

Comment:
Instead of raising ValueError, which is very generic; I'd rather define a new error type in this class. Something such as `class MultipleReleaseProductsError(ValueError)`.

This way if we get a `ValueError` for some other reason from a nested call, we won't display a potentially incorrect error message.